client/admin/api/user.js

```diff
@@ -23,9 +24,19 @@ function remove(item) {
   return request.delete(url.resource(item));
 }
 
+function bulkImport(items) {
+  return request({
+    method: 'post',
+    responseType: 'arraybuffer',
+    url: url.import,
+    data: items
+  });
```

Comment:
Use `request.post(url, config)` form :bowtie: 
You are expecting file as response? If so use `responseType: blob` instead 😉